tee-worker/local-setup/launch.py

```diff
@@ -164,6 +167,28 @@ def setup_environment(offset, config, parachain_dir):
             for flag in config["workers"][0]["flags"]
         ]
 
+def setup_worker_log_level():
```

Comment:
Maybe it's better to take the path as parameter

Reply:
Yeah, much better to set it as parameter. @zhouhuitian